db/migrate/20210121111312_change_responsible_body_who_can_order.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeResponsibleBodyWhoCanOrder < ActiveRecord::Migration[6.0]
+  def up
+    ResponsibleBody.where(who_will_order_devices: 'schools').update_all(who_will_order_devices: 'school')
+  end
+
+  def down
+    ResponsibleBody.where(who_will_order_devices: 'school').update_all(who_will_order_devices: 'schools')
```

Comment:
I'd delete this line and make it non-reversible. once its melded in you can't take it apart

Reply:
Good idea, thanks.